packages/app-portal/src/systems/Core/utils/url.ts

```diff
@@ -4,7 +4,12 @@ const trimPath = (path = '') => path.replace(trimRegex, '');
 
 export const getUrlHostName = (url: string) => {
   try {
-    return new URL(url || 'https://#').hostname;
+    const { hostname } = new URL(url || 'https://#');
+    if (hostname.startsWith('www.')) {
+      return hostname.replace('www.', '');
+    }
+
+    return hostname;
```

Comment:
Looks good but it can be shortened
```suggestion
    return hostname.replace(/^www\./,'');
```

Reply:
Thanks, I'll keep it in my notes for a following PR. 
Auto-merge was enabled.